Gemfile

```diff
@@ -62,6 +62,7 @@ gem "woothee"
 gem "sentry-rails"
 gem "scout_apm"
 
+gem "jb"
```

Comment:
We're using `jb` only for `image_path` which can be replaced with simple strings?

Reply:
Possible.

Reply:
0189854b538f